src/waylandhelper.h

```diff
@@ -0,0 +1,38 @@
+#include <QtCore>
+
+#include <KWayland/Client/connection_thread.h>
+#include <KWayland/Client/registry.h>
+#include <KWayland/Client/surface.h>
+#include <KWayland/Client/xdgforeign.h>
+
+using namespace KWayland::Client;
+
+#ifndef WAYLANDHELPER_H
+#define WAYLANDHELPER_H
```

Comment:
Include guards should be always defined above the everything else. This is a mechanism to avoid double inclusion.